### DISCUSSION TITLE
Excalifont: add support for Kyrgyz characters

### DISCUSSION THREAD

**Discussion by strvbkzhn** [excalidraw/excalidraw#8810] [General]
It would be great if support for Kyrgyz characters could be added. Kyrgyz uses the Cyrillic script with a few additional letters: `Өө`, `Үү`, and `Ңң`.

[More about Kyrgyz language](https://en.wikipedia.org/wiki/Kyrgyz_language)

**Comment by Mrazator** [comment]
Thanks, noted! Let's track it here https://github.com/excalidraw/excalidraw/issues/8838
